lesson_02/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+	"github.com/raydeth/gocourse/lesson_02/math"
+)
+
+func main() {
+	math.Fibonacci(10)
+	math.FmtFibonacci(15, "Число: %d\n")
+
+	defer fmt.Println("Finished")
```

Comment:
You should use `defer` at the beginning of the function - it doesn't make sense to defer something which is already at the very bottom